docs/swupd.1

```diff
@@ -131,7 +125,14 @@ the listed bundles, either directly or indirectly\.
 .
 .nf
 
-List all installed sofware bundles in the local system\.
+List all installed sofware bundles in the local system\. Bundles available
+can be listed with the `\-\-all` option\.
+
+*  `\-a, \-\-all`
+
+    Lists all available software bundles, either installed or not, that
+    are available\. It will return 0 with succeeded and a different value
+    of 0 with failed.
```

Comment:
This is a great step in documenting functionality specifying the cases when return is success or failure and the meaning, this should be rather be propagated to the rest of sub-commands man pages.

I also rather prefer to put an specific section in man page called "EXIT STATUS" to provide this information